Makefile

```diff
@@ -366,18 +363,14 @@ test.integration.kind:
 		CP="kind"
 
 .PHONY: test.e2e.gke
-test.e2e.gke:
-	CLUSTER_NAME="e2e-$(uuidgen)" \
-		KUBERNETES_CLUSTER_NAME="${CLUSTER_NAME}" go run hack/e2e/cluster/deploy/main.go \
-		KONG_TEST_CLUSTER="gke:${CLUSTER_NAME}" \
+test.e2e.gke: gotestsum
+	KONG_TEST_CLUSTER="gke:${CLUSTER_NAME}" \
```

Comment:
I think that it might not work without `go run hack/e2e/cluster/deploy/main.go`. Our tests do not use KTF to spin up fresh GKE clusters (see https://github.com/Kong/kubernetes-ingress-controller/blob/4bfe403e45e388e3d725aa9b8436e6c6da90e079/test/e2e/helpers_test.go#L68).